src/BlazorUI/Bit.BlazorUI.Extras/Styles/extra-components.scss

```
@@ -11,5 +11,6 @@
 @import "../Components/MessageBox/BitMessageBox.scss";
 @import "../Components/NavPanel/BitNavPanel.scss";
 @import "../Components/PdfReader/BitPdfReader.scss";
+@import "../Components/PhoneInput/BitPhoneInput.scss";
```

Stylelint (17.12.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)